evm/src/main/java/org/hyperledger/besu/evm/gascalculator/CancunGasCalculator.java

```diff
@@ -42,18 +58,34 @@ public long getTransientStoreOperationGasCost() {
   }
 
   @Override
-  public long dataGasCost(final int blobCount) {
+  public long dataGasUsed(final int blobCount) {
     return DATA_GAS_PER_BLOB * blobCount;
   }
 
-  @Override
-  public long computeExcessDataGas(final long parentExcessDataGas, final int newBlobs) {
-    final long consumedDataGas = dataGasCost(newBlobs);
-    final long currentExcessDataGas = parentExcessDataGas + consumedDataGas;
+  /**
+   * Retrieves the target data gas per block.
+   *
+   * @return The target data gas per block.
+   */
+  public long getTargetDataGasPerBlock() {
+    return TARGET_DATA_GAS_PER_BLOCK;
```

Comment:
To be clear, this is here because the target gas per block is baked into the protocol and doesn't float like the block gas limit?

Reply:
Yep, thats correct